Project.toml

```diff
@@ -8,9 +8,9 @@ Requires = "ae029012-a4dd-5104-9daa-d747884805df"
 StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 
 [compat]
-julia = "1.1"
 Requires = "0.5.2"
-StaticArrays = "0.11.1"
+StaticArrays = "0.12"
```

Comment:
It should probably keep 0.11 though?

Reply:
Why?

Reply:
Cause your package works with StaticArrays@0.11?

Reply:
Ok, that makes sense.  So, this needs an issue in CompatHelper.

Reply:
Yea

Reply:
https://github.com/bcbi/CompatHelper.jl/issues/31